src/PassportServiceProvider.php

```diff
@@ -119,6 +126,28 @@ protected function buildAuthCodeGrant()
     }
 
     /**
+     * Create and configure an instance of the Implicit grant.
+     *
+     * @return ImplicitGrant
+     */
+    protected function makeImplicitGrant()
+    {
+        return $this->buildImplicitGrant();
+    }
+
+    /**
+     * Build the Implicit grant instance.
+     *
+     * @return ImplicitGrant
+     */
+    protected function buildImplicitGrant()
```

Comment:
I think `makeImplicitGrant` and `buildImplicitGrant` method sound similar! Can they be merged?